### ISSUE TITLE
XML Error on Manually Check for Updates after opening Komodo.

### ISSUE THREAD

**Issue by codermjb** [Komodo/KomodoEdit#3234]
### Short Summary
First time you check for updates after every open says malformed XML: http://prntscr.com/gt9na2

### Steps to Reproduce
Check for updates after opening Komodo (From Menu -> Help -> Check for Updates)

### Expected results
Updates get checked

### Actual results
XML Error http://prntscr.com/gt9na2

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 10 and 11.0.1 nightly
*Operating System (and version)?* Xubuntu 16.0.4.2 LTS

**Comment by Naatan** [comment]
Can you reproduce this?

**Comment by codermjb** [comment]
Yes, 100 percent of the time after opening Komodo and running the update for the first time. So it always requires the second check.

**Comment by mitchell-as** [comment]
This happens every now and then and tends to sort itself out over time: #2539

**Comment by codermjb** [comment]
What should I do about the issue? It happens 100 percent of the time ever since Komodo and it doesn't check for updates on both computers. It's nothing major, it is just annoying having to go through the process of checking updates 2x all the time.

**Comment by Naatan** [comment]
You're on a nightly build right?

**Comment by codermjb** [comment]
@Naatan yes, just recently, but it happens to Komodo 10 and 11 non nigthly builds (I never used nightly builds before), it happens on the first nightly build and 2nd nightly build that were recently released as well.

Trying to get a screen capture.

**Comment by codermjb** [comment]
[xml-update-error.ogv.zip](https://github.com/Komodo/KomodoEdit/files/1356906/xml-update-error.ogv.zip)

@Naatan
